native/cocos/core/scene-graph/Node.h

```diff
@@ -663,6 +675,10 @@ class Node : public BaseNode {
 
     Mat4 _worldMatrix{Mat4::IDENTITY};
 
+    /* set _flagChangeVersion to globalFlagChangeVersion when `_flagChange` updated.
+    * `globalFlagChangeVersion == _flagChangeVersion` means tha "_flagChange is dirty in current frametime".
+    */
+    uint32_t _flagChangeVersion{0};
     uint32_t _flagChange{0};
```

Comment:
I think should rename to `_hasChangedFlags ` as TS.

Reply:
Updated